### PR TITLE
Feature/20 Add linter, prettier, pre-push linter. Reformat all files.

### PR DESCRIPTION
**Pull request by Kartm**

### CODE REVIEW

File: frontend-react/src/App.tsx [hunk 1]
```diff
@@ -1,88 +1,83 @@
...
+    axios.get(`/plan-it/user/getAllEmails/${id}`).then((response) => setAccountEmails(response.data));
+  };
 
-    // @ts-ignore
-    const login = useGoogleLogin({
-        onSuccess: onSuccess,
-        flow: "auth-code",
-        scope:
-            "profile email openid https://www.googleapis.com/auth/calendar https://www.googleapis.com/auth/calendar.readonly",
-        accessType: "offline",
```
Comment (Kaspr-whtev) [inline frontend-react/src/App.tsx:66]: Why get rid of accessType: "offline"?

**Review by Kaspr-whtev — CHANGES_REQUESTED** [review]

File: frontend-react/src/App.tsx (hunk 1 above)
Comment (Kartm) [reply frontend-react/src/App.tsx:66]: When removed `@ts-ignore`, I found out that TypeScript didn't like `accessType` field being set there. After a bit of research it turned out that there isn't such a parameter in the documentation at all: https://www.npmjs.com/package/@react-oauth/google

Therefore I removed it, and after successfully testing the auth flow, I decided it's redundant and it's safe to remove.

Comment (Kaspr-whtev) [reply frontend-react/src/App.tsx:66]: i guess it was a hold over from the previous library to interact with google

**Review by Kaspr-whtev — APPROVED** [review]

File: frontend-react/src/App.tsx (hunk 1 above)
Comment (Kartm) [reply frontend-react/src/App.tsx:66]: yeah

**Pull request merged**